frontend-react/src/webreceiver-utils.ts

```diff
@@ -24,4 +24,23 @@ const permissionCheck = (permission: String, authState: AuthState) => {
 // A receiver is anyone with an organization that is not "DHSender", i.e.: "DHaz_phd"
 const reportReceiver = (authState: AuthState) => {return authState.accessToken?.claims.organization.find(o => !o.includes(PERMISSIONS['sender']))};
```

Comment:
It will be nice to eventually get some unit tests on utility functions like the ones in this file. 
... but this is NOT blocking this PR.